components/link-carousel.js

```diff
@@ -53,7 +53,7 @@ export const LinkCarousel = ({ links }) => {
             }}
```

Comment:
This comment is actually for line 48 - the onMouseEnter means the link carousel only updates the image for a mouse/pointer interaction, rather than a Tab. Is there a way for us to update this to work for a keyboard as well as a mouse?

Reply:
I've added a event listener for when the links are focused so the image updates on keyboard interaction